### ISSUE TITLE
suggestion for , amibiguous name between drawing.Image  vs facerecognitionnet.image  ,

### ISSUE THREAD

**Issue by blackholeearth** [takuya-takeuchi/FaceRecognitionDotNet#179]
i have this  
 amibiguous name between drawing.Image  vs  facerecognitionnet.image  ,
 amibiguous name between drawing.rectance  vs facerecognitionnet.rectange ,
....

FRDN Image or Rectangle is not System.Drawing .  isnt it??
 * we cannot assing them directly ,  they have different structure. they have different purposes.

* because this entities/classes belongs to this library *FRDN*.

i think it would be better to rename these classes like this: ( which creastes amibigious messages in visual studio.)

 *  FRDN.Image    to  FRImage 
 * FRDN.rectangle to FRRectangle 
 *  others that i dont know  etc....

**Comment by takuya-takeuchi** [comment]
> FRDN Image or Rectangle is not System.Drawing . isnt it??

Yes, These Image and Rectangle are in FaceRecognitionDotNet namespace.

> i think it would be better to rename these classes like this

Thanks for your propose.
But I will not rename this because I want to simplify library name.

**Comment by blackholeearth** [comment]
C#.  **Namespace alias**   fixes ambiguity error.

İ dont have to explicitly Write  fullNamespace  everytime .


````
Using System; 
Using System.Drawing;
.....
using FRImage = FaceRecognitionDotNet.Image;
using FRRect = FaceRecognitionDotNet.Rectangle;

Class Test
{
      FRImage convert (Image  img)
      {
           // Do sth
          Return. New FRImage();
      }
}



```
